server/middleware/check-client-version.js

```diff
@@ -1,6 +1,8 @@
 import packageJSON from '../../package.json' with { type: "json" };
 const version = packageJSON.version;
 
+//This should be only for internal calls, but right now prevents spam api calls, this should be done with a proper cors policy
```

Comment:
This is not intended as a spam blocker. It's just checking that the version of Homebrewery on the client Browser matches the latest version, to avoid issues interacting with brews if things have changed between versions. If a cors policy is added to prevent spam API calls, this version check is still needed, but it could be moved to only apply to brew update calls.

Reply:
Ideally, either that or only apply to internal calls.